test/api/test_system.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2021-present ScyllaDB
+#
+# This file is part of Scylla.
+#
+# Scylla is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Scylla is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Scylla.  If not, see <http://www.gnu.org/licenses/>.
+
+def test_system_uptime_ms(cql, api):
```

Comment:
If this specific test doesn't need "cql", you don't need it in the argument list.
I still don't know if you plan any tests that will need CQL, but this one certainly doesn't.

Reply:
additional tests added later in the series do use it.

Reply:
Ok, I understand that now, but in this case, this specific test "test_system_uptime_ms" doesn't need the cql fixture, so it shouldn't list it.